src/wp-admin/themes.php

```diff
@@ -555,7 +555,7 @@
 			?>
 			<a class="button activate" href="<?php echo $theme['actions']['activate']; ?>" aria-label="<?php echo esc_attr( $aria_label ); ?>"><?php _e( 'Activate' ); ?></a>
 			<?php
-			if ( ! $theme['blockTheme'] && current_user_can( 'edit_theme_options' ) && current_user_can( 'customize' ) ) {
+			if ( current_user_can( 'edit_theme_options' ) && current_user_can( 'customize' ) ) {
```

Comment:
Following up on https://github.com/WordPress/wordpress-develop/pull/4627#discussion_r1232534915, since you mention that the `customize` capability doesn't apply to block themes, I think we could improve the clause here: It needs to apply to classic themes (since they use the Customizer), so we have to keep it, but we could exclude it when using a block theme:
```suggestion
			// Only classic themes require the "customize" capability.
			if ( current_user_can( 'edit_theme_options' ) && ( $theme['blockTheme'] || current_user_can( 'customize' ) ) ) {
```